arc/runner/common-values.yaml

```diff
@@ -0,0 +1,32 @@
+githubConfigUrl: "https://github.com/utilitywarehouse"
+
+githubConfigSecret: arc-auth
+
+minRunners: 1
+
+containerMode:
+  type: "kubernetes"
+  kubernetesModeWorkVolumeClaim:
+    accessModes: ["ReadWriteOnce"]
+    storageClassName: "netapp-ontap-san-ext4"
+    resources:
+      requests:
+        storage: 5Gi
+
+template:
+  spec:
+    # https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/troubleshooting-actions-runner-controller-errors#error-access-to-the-path-homerunner_work_tool-is-denied
+    securityContext:
```

Comment:
some workflows use sudo, would this allow for that?

Reply:
yes, sudo can be used inside the image, but the container is not privileged, so cannot be used to escape it